finagle-http/src/test/scala/com/twitter/finagle/http/StatusTest.scala

```diff
@@ -15,4 +16,45 @@ class StatusTest extends FunSuite {
     assert(Status(599).reason == "Server Error")
     assert(Status(601).reason == "Unknown Status")
   }
+
+  test("Pattern match usability check") {
+    null match {
```

Comment:
Right, but my impression is that in scala we typically assume that null is not a valid argument to pass.  If an instance might be null, it's up to the user to use it safely.  If that weren't the case, we would need to null-check every argument ever.

We could make the argument that there's no harm in adding a test for null, but since folks often look at tests as example code, I'd be worried that this will end up being cargo-culted.